rules/defaults.yaml

```diff
@@ -159,6 +159,10 @@
 		Range: 4c0
 	Tooltip:
 		Description: Tech Building
+	CaptureNotification:
+		Notification: TechBuildingCaptured
+	ActorLostNotification:
```

Comment:
Does this play when actor is destroyed? In RA2 this only plays when anyone else captures our Tech Building iirc.